Authentication/Authentication/Program.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.Metadata.Ecma335;
+
+namespace Authentication
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var path = @"./Data/Users.txt";
+            string choice = "0";
+
+            CheckDuplicateUsers(path);
+
+            while (!choice.Equals("Exit", StringComparison.OrdinalIgnoreCase))
+            {
+                choice = GetUserChoice();
+                if (choice.Equals("Login", StringComparison.OrdinalIgnoreCase))
```

Comment:
Instead of repeating the same capitalization ignoring compare, make choice lowercase and then you will even be able to use a `switch` statement.

Reply:
Ok, I thought that the menu system should accept all types of capitalisation disregarding the capitalisation (whether a user users Login or login or LOGIN or LoGiN) just as a fail safe mechanism.